doc/classes/String.xml

```diff
@@ -530,13 +530,13 @@
 				Performs a case-insensitive comparison to another string. Returns [code]-1[/code] if less than, [code]+1[/code] if greater than, or [code]0[/code] if equal.
 			</description>
 		</method>
-		<method name="ord_at">
+		<method name="unicode_at">
 			<return type="int">
 			</return>
-			<argument index="0" name="at" type="int">
+			<argument index="0" name="idx" type="int">
 			</argument>
 			<description>
-				Returns the character code at position [code]at[/code].
+			        Returns the character code at position [code]idx[/code].
```

Comment:
You changed the tab to spaces.

Reply:
sorry clang-formate doesn't work for me on Windows. I will fix it when I get home

Reply:
`clang-format` doesn't apply to the XML, for those you need to preserve the formatting manually.